pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, nmap, jq, cifs-utils, sshfs, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+
+  pname   = "rmount";
+  version = "1.0.1";
+  name    = "rmount-${version}";
```

Comment:
I should have clarified on the original PR: with `pname` and `version` set, `name` is no longer required, as name is always declared as `name = "${pname}-${version}";`